src/net.h

```diff
@@ -1498,6 +1514,7 @@ class CConnman
     int m_max_outbound_block_relay;
 
     int m_max_addnode{MAX_ADDNODE_CONNECTIONS};
+    int m_max_private_broadcast{MAX_PRIVATE_BROADCAST_CONNECTIONS};
```

Comment:
In 0ba30dec085f860c89f4fd8c8a398aa8a13b5ebd "net: implement opening PRIVATE_BROADCAST connections": `m_max_private_broadcast` is not used.

Reply:
Right! Removed.